include/zephyr/usb/usbd.h

```diff
@@ -200,6 +200,8 @@ struct usbd_status {
 	unsigned int rwup : 1;
 	/** USB device speed */
 	enum usbd_speed speed : 2;
+	/** USB device support enable operation is suspended */
```

Comment:
This is very confusing name. Do not use USB terms (here: suspend) in non-USB specification meaning.